enterprise-suite/es-grafana/kubernetes-exporter.json

```diff
@@ -1,7 +1,38 @@
 [
-   {"graphName": "CPU Percent", "promQL":["node_cpu_percent"]},
-   {"graphName": "Memory Available Percent", "promQL":["node_memory_usable_percent"]},
-   {"graphName": "File System Free Percent", "promQL":["node_filesystem_free_percent"]},
-   {"graphName": "Bytes Received Per Second", "promQL":["sum by (instance) (irate(node_network_receive_bytes[1m]))"]},
-   {"graphName": "Bytes Transmitted Per Second", "promQL":["sum by (instance) (irate(node_network_transmit_bytes[1m]))"]}
+    {
+        "graphName": "CPU Percent",
+        "promQL": [
+            "sum without(container_name,id,image,name)((irate(container_cpu_user_seconds_total{ContextTags}[1m]) + irate(container_cpu_system_seconds_total[1m])) * 100)"
+        ]
+    },
+    {
+        "graphName": "Active Memory Used Percent",
+        "promQL": [
+            "sum without(container_name,id,image,name)((container_memory_working_set_bytes{ContextTags} / container_spec_memory_limit_bytes < +Inf or container_memory_working_set_bytes{ContextTags} / on (node_ip) group_left node_memory_MemTotal) * 100)"
+        ]
+    },
+    {
+        "graphName": "Total Memory Used Percent",
```

Comment:
Should we mention here this is out of the limit or out of the node? Can we set a graph description perhaps?

Reply:
There's nothing in the current config to let us do that, but I have a change coming that would let us set the legend with a template.  Not sure if it can do an expression or just label swaps..would have to dig a little.  We can / should certainly document our graphs.